src/scripts/main.js

```diff
@@ -1,3 +1,187 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const rows = tbody.querySelectorAll('tr');
+const form = document.createElement('form');
+
+let count = 1;
+
+const selected = (e) => {
+  const target = e.target;
+  const selectTr = e.target.parentNode;
+
+  if (target.closest('tr')) {
+    selectTr.classList.add('active');
+  }
+};
+
+form.classList.add('new-employee-form');
+
+form.insertAdjacentHTML('afterbegin', `<form action="#" method="GET">
+<label for="name">
+  Name:
+  <input name="name" data-qa="name" type="text" required>
+</label>
+<label for="position">
+  Position:
+  <input name="position" data-qa="position" type="text" required>
+</label>
+<label for="office">
+  Office:
+  <select name="office" data-qa="office" id="office" required>
+    <option value="Tokyo">Tokyo</option>
+    <option value="Singapore">Singapore</option>
+    <option value="London">London</option>
+    <option value="New York">New York</option>
+    <option value="Edinburgh">Edinburgh</option>
+    <option value="San Francisco">San Francisco</option>
+  </select>
+</label>
+<label for="age">
+  Age:
+  <input name="age" data-qa="age" type="number" required>
+</label>
+<label for="salary">
+  Salary:
+  <input name="salary" data-qa="salary" type="number" required>
+</label>
+<button type="submit">Save to table</button>
+</form>`);
```

Comment:
```suggestion
form.insertAdjacentHTML('afterbegin', `
<form action="#" method="GET">
<label for="name">
  Name:
  <input name="name" data-qa="name" type="text" required>
</label>
<label for="position">
  Position:
  <input name="position" data-qa="position" type="text" required>
</label>
<label for="office">
  Office:
  <select name="office" data-qa="office" id="office" required>
    <option value="Tokyo">Tokyo</option>
    <option value="Singapore">Singapore</option>
    <option value="London">London</option>
    <option value="New York">New York</option>
    <option value="Edinburgh">Edinburgh</option>
    <option value="San Francisco">San Francisco</option>
  </select>
</label>
<label for="age">
  Age:
  <input name="age" data-qa="age" type="number" required>
</label>
<label for="salary">
  Salary:
  <input name="salary" data-qa="salary" type="number" required>
</label>
<button type="submit">Save to table</button>
</form>
`);
```
еще нежуно добавть отступы от края